File: typescript/basic/src/ejemplos/casos/ejemplo02.ts
let var13: string;

var13 = prompt("digite el numero");

switch(var13){
    case "uno":
    case "UNO":{
        console.log(1);
        break;
    }
    case "dos":
    case "DOS":{
        console.log(2);
        break;
    }
    case "tres":
    case "TRES":{
        console.log(3);
        break;
    }
    default:{
        console.log("no conozco ese numero");
    }

}

File: java/javabasico/src/main/java/co/edu/sena/javabasico/casos/Ejemplo02.java
package co.edu.sena.javabasico.casos;

import java.util.Scanner;

public class Ejemplo02 {
    public static void main(String[] args) {
        String b;
        Scanner teclado = new Scanner(System.in);
        b = teclado.next();

        switch (b){
            case "uno":
            case "UNO":
                System.out.println(1);
                break;
            case "dos":
            case "DOS":
                System.out.println(2);
                break;
            case "tres":
            case "TRES":
                System.out.println(3);
                break;
            default:
                System.out.println("no conozco ese numero");
        }

    }
}
